#current position of client
geographic_msgs/GeoPoint position

#current orientation based on compass North(degree)
float64 angle